node/subsystem-test-helpers/src/lib.rs

```diff
@@ -321,7 +324,7 @@ pub fn subsystem_test_harness<M, OverseerFactory, Overseer, TestFactory, Test>(
 
 	futures::executor::block_on(async move {
 		future::join(overseer, test)
-			.timeout(Duration::from_secs(2))
+			.timeout(Duration::from_secs(8))
```

Comment:
Doc comment above needs changing, and also, I'm interested in the motivation of this change.

Reply:
2 seconds was too low for dispute distribution tests to pass. (Crypto APIs in debug are awfully slow.)